features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/impl/DatadogTracing.kt

```diff
@@ -52,62 +39,49 @@ object DatadogTracing {
      * configurations or features are unavailable.
      */
     fun newTracerBuilder(sdkCore: SdkCore = Datadog.getInstance()): DatadogTracerBuilder = when {
-        builderProvider != null -> builderProvider as DatadogTracerBuilder
-        sdkCore !is FeatureSdkCore -> NoOpDatadogTracerBuilder()
+        DatadogTracingInternalToolkit.builderProvider != null -> {
+            DatadogTracingInternalToolkit.builderProvider as DatadogTracerBuilder
+        }
+        sdkCore !is FeatureSdkCore -> {
+            NoOpDatadogTracerBuilder()
```

Comment:
maybe we need to log a warning here?

Reply:
if `sdkCore` is not an `FeatureSdkCore` we don't have an access to `internalLogger`